cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -247,4 +247,6 @@ type AutoscalingOptions struct {
 	ParallelDrain bool
 	// NodeGroupSetRatio is a collection of ratios used by CA used to make scaling decisions.
 	NodeGroupSetRatios NodeGroupDifferenceRatios
+	// ExternalNodeDeletion configures CA to let the external cloud providers handle deletion of nodes.
```

Comment:
This comment does not explain when or why a user might want to enable this setting. We should expand this to clarify why you might want to enable this and what providers support it right now (clusterapi).